sdk/core/src/policy.rs

```diff
@@ -1,58 +1,15 @@
-use crate::{Request, Response};
-
-use async_trait::async_trait;
-
+use crate::retry_policy::RetryPolicy;
+use crate::Request;
+use crate::Response;
```

Comment:
Nit: why did we break this into two lines?

Reply:
No reason! Will fix!